src/installer/managed/Microsoft.NET.HostModel/Microsoft.NET.HostModel.csproj

```diff
@@ -21,14 +21,9 @@
   <ItemGroup>
     <!-- SDK pins this to a lower version https://github.com/dotnet/sdk/issues/43325 -->
     <PackageReference Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataToolsetVersion)" />
```

Comment:
Is this one possible to replace too?  I know we haven't had a CVE in this, but it would be nice if we could make all of HostModel's dependencies live.

Reply:
Filed https://github.com/dotnet/runtime/issues/112134